one_fm/processor.py

```diff
@@ -10,6 +10,11 @@ def sendemail(recipients, subject, header=None, message=None,
         content=None, reference_name=None, reference_doctype=None,
         sender=None, cc=None , attachments=None, delay=None):
     logo = "https://one-fm.com/files/ONEFM_Identity.png"
+    
+    if attachments:
+            message += """
+            <p>Please find the attached Proof Document in the mail below.</p>
```

Comment:
Is the attachment is always a proof document ?!

Reply:
Yes, for leave application, it's always gonna be proof_document. We do not have any other attachment field.